lib/src/ConfigLoader.cc

```diff
@@ -393,6 +393,12 @@ static void loadApp(const Json::Value &app)
     {
         drogon::app().enableRunAsDaemon();
     }
+    // handle SIGTERM
+    auto handleSigterm = app.get("handle_sig_term", true).asBool();
+    if (handleSigterm)
```

Comment:
I think this line should be `if(!handleSigterm)`

Reply:
Oops !